flask/app/models.py

```diff
@@ -421,25 +424,36 @@ class Variation(str, Enum):
     Stop_Lost = "Stop_Lost"
     Start_lost = "Start_lost"
     Protein_Altering_Variant = "Protein_Altering_Variant"
-    Start_Retrained_Variant = "Start_Retrained_Variant"
+    Start_Retained_Variant = "Start_Retained_Variant"
     Mature_miRNA_Variant = "Mature_miRNA_Variant"
-    Stop_Retained_Variant = "Stop_Retained_Variant"
 
 
+# gene can have many variants
+# one to many
+#
+
+
+@dataclass
 class Gene(db.Model):
     gene_id: int = db.Column(db.Integer, primary_key=True)
-    hgnc_gene_id: int = db.Column(db.Integer)
-    ensembl_id: int = db.Column(db.Integer)
+    hgnc_gene_id: int = db.Column(db.Integer, unique=True)
+    ensembl_id: int = db.Column(db.Integer, unique=True)
+    gene: str = db.Column(db.String(50))
     hgnc_gene_name: str = db.Column(db.String(50))
+    variant = db.relationship("Variant", backref="gene")
```

Comment:
Might be misunderstanding something but I don't think it would be possible to have the relationship on the variant (child) table? I'm looking at the documentation and it states that for a one-to-many relationship the relationship must be created through the parent